build.gradle.kts

```diff
@@ -18,16 +18,16 @@ import org.jetbrains.kotlin.gradle.dsl.KotlinProjectExtension
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
-  kotlin("jvm") version versions.kotlin apply false
-  id("org.jetbrains.dokka") version versions.dokka apply false
-  id("com.diffplug.spotless") version versions.spotless
-  id("com.vanniktech.maven.publish") version versions.mavenPublish apply false
+  kotlin("jvm") version libs.versions.kotlin.get() apply false
```

Comment:
This DSL is truly horrible

Reply:
Indeed, was thinking to look for alternatives

Reply:
`classpath` thems is a better way ?

Reply:
@Goooler agreed, this is much nicer! #1130